api/external/compliance/reporting/reporting.proto

```diff
@@ -99,6 +99,28 @@ service ReportingService {
 		};
 	};
 
+	/*
+	List Control info
+	Lists controls from the last run, with optional filtering.
+	Supports filtering and pagination. Maximum 100 search can be 
+	made when specifying the pagination from and size. Sum of from+size
+	should be less that 100. By default 10 results will be returned.
```

Comment:
typo: it should be `less than 100`

Reply:
created https://github.com/chef/automate/issues/5888 to fetch results beyond 100 results